install.sh

```diff
@@ -114,124 +193,152 @@ else
   fi
 fi
 
-rosdep -q update
 echo -e "\e[32mDone: Configuring rosdep\e[0m"
 echo ""
 
-
-echo -e "\e[94mConfiguring Clearpath Setup\e[0m"
```

Comment:
We still need to check for / create the folder and set up the user rights one way or another, is this functionality being moved elsewhere?

Reply:
Re-added.